### PR TITLE
Fix missing CSS mixing and variables issue in root storybook

### PR DESCRIPTION
**Pull request by wojtekn**
Related to p1684333670968859-slack-C02DQP0FP

## Proposed Changes

In this PR, I propose to fix the error that pops out when I start the root storybook and prevents me from using it. I fix it by adding an import for files that define breakpoint mixins and typography variables.

## Testing Instructions

1. Run storybook in root:
```
yarn storybook:start
```

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by kozer — APPROVED** [review]

**Review by okmttdhr — APPROVED** [review]

File: packages/design-picker/src/components/theme-card/style.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import "@automattic/typography/styles/variables";
+@import "client/assets/stylesheets/shared/mixins/_breakpoints";
```
Comment (taipeicoder) [inline packages/design-picker/src/components/theme-card/style.scss]: Hi @wojtekn! As this P2 p4TIVU-9lC-p2 mentions, we try to keep packages separated from Calypso client. @okmttdhr I'm not familiar to how Storybook works, but is it possible to import these files on Storybook's side?

File: client/components/card-heading/style.scss [hunk 2]
```diff
@@ -1,3 +1,6 @@
+@import "@automattic/typography/styles/variables";
+@import "../../assets/stylesheets/shared/mixins/_breakpoints";
```
Comment (taipeicoder) [inline client/components/card-heading/style.scss]: Could we use `@import "calypso/assets/stylesheets/shared/mixins/breakpoints";` instead as suggested in p4TIVU-9sM-p2 ? Thank you!

File: packages/design-picker/src/components/theme-card/style.scss (hunk 1 above)
Comment (wojtekn) [reply packages/design-picker/src/components/theme-card/style.scss]: My bad, `_breakpoints.scss` was not needed in this file.

File: client/components/card-heading/style.scss (hunk 2 above)
Comment (wojtekn) [reply client/components/card-heading/style.scss]: Thanks for the suggestion, changed.

**Review by taipeicoder — APPROVED** [review]
LGTM, thanks for the addition 👍

**Review by taipeicoder — APPROVED** [review]
LGTM, thanks for the addition 👍

**Pull request merged**